src/civ2.js

```diff
@@ -32,17 +31,31 @@ const civ2 = require('./lib/civ2-commands');
 const routes = require('./lib/routes');
 const gh = require('./lib/github');
 const aws = require('./lib/aws');
+const { log, error } = require('./lib/utils');
+const PivotalTracker = require('./lib/pivotal-tracker');
 
 const REPLICATED_STABLE_CHANNEL = 'Stable';
 const DEMO = { url: 'demo.stoic.cc', name: 'demo' };
-const parsedPrs = new Map();
+const CHANNELS = {
+  default: '#testing-ci',
+  releaseCandidates: '#release-candidates'
+};
+const MAX_PRDATE = 1000 * 60 * 5;
```

Comment:
`MAX_PR_DATE` would be easier to read, I thought it was a typo for `MAX_UPDATE` at first :)
Also, don't use DATE as suffix, prefer the real unit for the constant. MINS? SECS? It would document the computation.